adapters/dmx/usersync_test.go

```diff
@@ -0,0 +1,19 @@
+package dmx
+
+import (
+	"testing"
+	"text/template"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDmxSyncer(t *testing.T) {
+	temp := template.Must(template.New("sync-template").Parse("https://cdn.districtm.io/ids/?sellerid=10007"))
```

Comment:
We've updated the template for this test. Did you get this old style from a doc we forgot to update?